recipes-domd/domd-image-weston/domd-image-weston.bb

```diff
@@ -0,0 +1,14 @@
+SUMMARY = "Image with weston for driver domain"
+
+LICENSE = "MIT"
+
+require recipes-dom0/dom0-image-weston/dom0-image-weston.bb
```

Comment:
I don't think, that we should rely on dom0 receipt. Who knows, what will change in dom0 recipe in the future. 
Also, it is rather strange to include .bb file into another .bb file.

I would recommend to move all shared code into separate .inc file and then include it in Dom0, DomU and DomD recipes.